server/config/config.go

```diff
@@ -54,10 +54,11 @@ type Config struct {
 	AdvertiseClientUrls string `toml:"advertise-client-urls" json:"advertise-client-urls"`
 	AdvertisePeerUrls   string `toml:"advertise-peer-urls" json:"advertise-peer-urls"`
 
-	Name              string `toml:"name" json:"name"`
-	DataDir           string `toml:"data-dir" json:"data-dir"`
-	ForceNewCluster   bool   `json:"force-new-cluster"`
-	EnableGRPCGateway bool   `json:"enable-grpc-gateway"`
+	Name                       string `toml:"name" json:"name"`
+	DataDir                    string `toml:"data-dir" json:"data-dir"`
+	ForceNewCluster            bool   `json:"force-new-cluster"`
+	EnableGRPCGateway          bool   `json:"enable-grpc-gateway"`
+	EnableLeaderClientAutoSync bool   `toml:"enable-leader-client-auto-sync" json:"enable-leader-client-auto-sync"`
```

Comment:
If this configuration cannot be changed online, it doesn’t seem to be of much use.

Reply:
This config needs to be restarted to take effect. It's hard to know when to set it.